test/parallel/test-http-common.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const httpCommon = require('_http_common');
+const checkIsHttpToken = httpCommon._checkIsHttpToken;
+const checkInvalidHeaderChar = httpCommon._checkInvalidHeaderChar;
+
+// checkIsHttpToken
+assert(checkIsHttpToken('t'));
+assert(checkIsHttpToken('tt'));
+assert(checkIsHttpToken('ttt'));
+assert(checkIsHttpToken('tttt'));
+assert(checkIsHttpToken('ttttt'));
+
+assert.strictEqual(checkIsHttpToken(' '), false);
+assert.strictEqual(checkIsHttpToken('あ'), false);
+assert.strictEqual(checkIsHttpToken('あa'), false);
+assert.strictEqual(checkIsHttpToken('aaaaあaaaa'), false);
+
+// checkInvalidHeaderChar
+assert.strictEqual(checkInvalidHeaderChar(1), false);
```

Comment:
Nit: How about adding the empty string case? It should slightly increase the coverage.